### DISCUSSION TITLE
Done with the Labs assignment, any other thing to be able to get my certificate

### DISCUSSION THREAD

**Discussion by kangahdesmond** [giswqs/geog-312#200] [Q&A]
Good day sir, I am Done with all the labs, is there any other thing for me to do to get my certificate

**Comment by giswqs** [comment]
Did you enroll in this course? The certificate will be sent automatically once all lab assignments are completed

https://tiny.utk.edu/intro-gis-programming

**Comment by kangahdesmond** [comment]
Thanks

Sent from my T-Mobile 5G Device
Get Outlook for Android<https://aka.ms/AAb9ysg>
________________________________
